mirage/utils.ts

```diff
@@ -1,7 +1,7 @@
 import faker from 'faker';
 
 export function placekitten(width: number, height: number, n = 12) {
-    return `https://placekitten.com/${width}/${height}?image=${n % 17}`;
+    return `https://placecats.com/${width}/${height}?image=${n % 17}`;
```

Comment:
they grow up so fast!

(tbh i'd rather remove all the various loads from third-party domains, but at least this one doesn't affect users 🐱 )